autoload/codefmt/zprint.vim

```diff
@@ -26,6 +26,30 @@
 
 let s:plugin = maktaba#plugin#Get('codefmt')
 
+""
+" @private
+function! s:ApplyToZprintOptions(opts, key, value) abort
+  call maktaba#ensure#IsList(a:opts)
+  call maktaba#ensure#IsString(a:key)
+
+  " We can only apply Vim settings for simple values of a:opts, because it's
+  " an EDN map, not a list of command-line flags.
+  let l:edn = empty(a:opts) ? '{}' : a:opts[0]
+  if l:edn[0] != '{'
+    return a:opts
+  endif
+
+  " Don't try to insert if key already appears in the EDN.
```

Comment:
It shouldn't try to override existing value for the key?

Reply:
You can't have duplicate keys in EDN, and replacing it would involve actually parsing the EDN, so I think it's better to defer in that case. But I'll handle this in a separate PR.